tests/OBSLocal.py

```diff
@@ -557,12 +557,16 @@ def create_submit_request(self, project, package, text=None):
         :type package: str
         :param text: commit message for initial package creation
         :type text: str
+        :param add_commit: whenever add initial package commit. Useful to disable
+               if package already exists
+        :type add_commit: bool
         :return: created request.
         :rtype: Request
         """
         project = self.create_project(project)
         package = Package(name=package, project=project)
-        package.create_commit(text=text)
+        if add_commit:
+            package.create_commit(text=text)
```

Comment:
the condition is neeed as check source checker otherwise complain about not mentioned file README created in this method at https://github.com/openSUSE/openSUSE-release-tools/pull/2648/files#diff-f26c9a37e9fec1909032289e143103fe74d3ca070d31b9512a0e14a51c1ae985R1008
I keep default true to avoid break of other tests